ee/ui/assets/assets.go

```diff
@@ -19,9 +19,21 @@ var MenubarCircleXLightmodeIco []byte
 //go:embed menubar-circle-x-lightmode.png
 var MenubarCircleXLightmodePng []byte
 
+//go:embed menubar-circle-x-monochrome-darkmode.ico
+var MenubarCircleXMonochromeDarkmodeIco []byte
+
+//go:embed menubar-circle-x-monochrome-darkmode.png
+var MenubarCircleXMonochromeDarkmodePng []byte
+
 //go:embed menubar-circle-x-monochrome.ico
 var MenubarCircleXMonochromeIco []byte
 
+//go:embed menubar-circle-x-monochrome-lightmode.ico
+var MenubarCircleXMonochromeLightmodeIco []byte
+
+//go:embed menubar-circle-x-monochrome-lightmode.png
+var MenubarCircleXMonochromeLightmodePng []byte
```

Comment:
I'm a little confused by this. I thought we didn't need the dark/light monochrome versions. And if we do have them, we do we still have `menubar-circle-x-monochrome` ?

Reply:
Hmm I'm not entirely sure, I just dropped the new set of icons in and hit generate. Maybe Fritz renamed some?

Reply:
To better answer your question - yes, for macOS we would only need one monochrome version which can be used as a template image for both dark and light modes. Not sure if Fritz had anything else in mind.

I can remove the `-monochrome-dark/light` ones and leave just the `-monochrome` ones if we don't see any need for them right now.

Reply:
I'd bet Fritz gave us more than we need

Reply:
(mostly I'm trying minimize the size we use)

Reply:
Removed  the `-monochrome-dark/light` ones, regenerated.